riocli/device/model.py

```diff
@@ -76,3 +104,54 @@ def validate(data):
         """
         schema = load_schema('device')
         schema.validate(data)
+
+    def create_hwil_device(self) -> str:
```

Comment:
todo: refactor and move these helper functions under `riocli/device/util.py` so that were can reuse them in `rio device create --virtual` command.